src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,114 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] elements;
+    private int position;
+
+    public ArrayList() {
+        elements = new Object[DEFAULT_CAPACITY];
+        position = -1;
```

Comment:
It will be better if we initialize it to 0, so it will be clearer for other developers.
```suggestion
        position = 0;
```